src/painting-annotations/helper.ts

```diff
@@ -92,7 +95,7 @@ export function createPaintingAnnotationsHelper(vault: CompatVault = compatVault
     return {
       types,
       items,
-      choice,
+      choice: choices.items.length < 2 ? choices.items[0] : choices,
```

Comment:
I think we also need to ensure the item is `null` to satisfy the types.
```suggestion
      choice: choices.items.length < 2 ? (choices.items[0] || null) : choices,
```

Reply:
thanks! Still getting my head around typescript!